libraries/c_sdk/standard/mqtt/test/unit/iot_tests_mqtt_api.c

```diff
@@ -630,6 +763,9 @@ TEST( MQTT_Unit_API, OperationCreateDestroy )
     /* Disconnect the MQTT connection, then call Wait to clean up the operation. */
     IotMqtt_Disconnect( _pMqttConnection, IOT_MQTT_FLAG_CLEANUP_ONLY );
     IotMqtt_Wait( pOperation, 0 );
+
+    /* Remove the MQTT Context for the MQTT connection. */
+    _IotMqtt_removeContext( _pMqttConnection );
```

Comment:
Wouldn't this be also part of the clean up? Do we have to explicitly remove it?

Reply:
Addressed